src/components/side_panel/chart/building_blocks/generic_side_panel/config_panel.ts

```diff
@@ -168,6 +281,83 @@ export class GenericChartConfigPanel extends Component<Props, SpreadsheetChildEn
     }
   }
 
+  get splitRanges(): CustomizedDataSet[] {
```

Comment:
```suggestion
  get spliitedRanges(): CustomizedDataSet[] {
```
?

Reply:
Split should be used instead of splitted :) 
![image](https://github.com/user-attachments/assets/2c572638-03d5-43ea-9a93-e6216e7f0d4a)